modules/framework/galasa-parent/dev.galasa.framework.k8s.controller/src/test/java/dev/galasa/framework/k8s/controller/TestPodSchedulerTest.java

```diff
@@ -88,7 +89,7 @@ private void assertPodDetailsAreCorrect(
         checkPodSpec(pod, settings);
     }
 
-    @SuppressWarnings("null")
+    // @SuppressWarnings("null")
```

Comment:
Why comment these out instead of just remove?

Reply:
Agreed. Wanted to see if things broke in the build. They didn't. Removed.